toolchain/check/import_ref.cpp

```diff
@@ -892,7 +892,8 @@ static auto GetLocalConstantId(ImportRefResolver& resolver,
     return SemIR::ConstantId::None;
   }
   auto import_decl_inst_id = resolver.import_generics().Get(generic_id).decl_id;
-  auto import_decl_inst = resolver.import_insts().Get(import_decl_inst_id);
+  auto import_decl_inst =
+      resolver.import_insts().GetWithAttachedType(import_decl_inst_id);
```

Comment:
FWIW, I don't think this change has any effect -- `GetLocalGenericId` doesn't care about the attachment of the constant that it's given, and is the only thing that should be consuming the constant returned by this function. I'd been trying to only call `GetWithAttachedType` where it's necessary, but maybe this case is subtle enough that it's clearer to just call it than to explain why it's not necessary :)

Reply:
Sure, that works for me. If we do eventually switch to storing attached types and constants out of band, we might want to revisit, but I think we'd want a broader reexamination of how we import attached types if we do that.